web/src/labs/marked/parser/InlineLatex.tsx

```diff
@@ -12,11 +12,7 @@ const inlineRenderer = (rawStr: string) => {
     } else if (matchResult[2]) {
       latexCode = matchResult[2];
     }
-    return (
-      <div className="w-full max-w-full overflow-x-auto">
-        <TeX key={latexCode}>{latexCode}</TeX>
-      </div>
-    );
+    return <TeX key={latexCode}>{latexCode}</TeX>;
```

Comment:
To prevent overflow above the x-axis.

```suggestion
    return <div className="max-w-full overflow-x-auto">
        <TeX key={latexCode}>{latexCode}</TeX>
      </div>
```